content/backmatter.tex

```diff
@@ -396,13 +396,14 @@ \subsection{\CorCpp: \_my\_pe, \_num\_pes, shmalloc, shfree, shrealloc, shmemali
 prefix for all routines.
 
 \subsection{\Fortran: START\_PES, MY\_PE, NUM\_PES}
-The \Fortran{} routines \FUNC{START\_PES}, \FUNC{MY\_PE}, and \FUNC{NUM\_PES}
+The \Fortran routines \FUNC{START\_PES}, \FUNC{MY\_PE}, and \FUNC{NUM\_PES}
 were deprecated in order to minimize the API differences from the deprecation
 of \CorCpp routines \FUNC{start\_pes}, \FUNC{\_my\_pe}, and \FUNC{\_num\_pes}.
 
 \subsection{\Fortran: SHMEM\_PUT}
-The \Fortran{} routine \FUNC{SHMEM\_PUT} is defined only for the \Fortran{}
-\ac{API} and is semantically identical to \Fortran{} routines
+The \Fortran function \FUNC{SHMEM\_PUT} is defined only for the \Fortran
+\ac{API} and is semantically identical to \Fortran functions
+
```

Comment:
@nspark Could you check on the change noted above?  Apart from that, I'm ready to merge.